PWGHF/D2H/Tasks/taskDirectedFlowCharmHadrons.cxx

```diff
@@ -0,0 +1,302 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file taskv1CharmHadrons.cxx
+/// \brief Analysis task for charm hadron flow
```

Comment:
Fix the name.
Why do you not use the existing charm flow task?
Be more specific about the difference w.r.t. the other charm flow task.

Reply:
[Reply]: Name is fixed now.
             This code is for v1 of D mesons where we need to use spectator plane. The earlier code was for v2 which uses the event plane. The physics and techniques are very much different so I prefer to use my own task and nt modify the existing one.